.eslint-dictionary.json

```diff
@@ -1,201 +1,394 @@
 [
+  "accessor",
+  "adaptor",
+  "adaptors",
+  "addgraphqldatasource",
+  "admingroup",
   "adminui",
+  "aggs",
+  "allowlist",
+  "alphanumerics",
   "amazonaws",
+  "amazoncognito",
+  "amd64",
+  "amplifyadmin",
+  "amplifyapp",
+  "amplifyconfig",
+  "amplifyconfiguration",
+  "amplifyconsole",
+  "amplifyhosting",
   "amplifymeta",
+  "amplifyrc",
+  "amplifytools",
+  "ansi",
   "apigateway",
   "apigw",
   "apis",
   "apns",
+  "appleid",
+  "appstate",
   "appsync",
   "archiver",
   "argv",
   "arn",
   "arns",
   "authorizer",
+  "auths",
   "authuser",
+  "autogenerated",
   "awaiter",
   "aws",
+  "awscfn",
   "awscloudformation",
+  "awscognitouserpool",
+  "awsconfig",
+  "awsconfiguration",
   "awsmobile",
   "axios",
   "babelrc",
   "backend",
+  "basedir",
+  "birthdate",
   "bitwise",
   "bool",
+  "browserify",
+  "browserslist",
+  "bugfix",
   "buildable",
   "buildspec",
   "callout",
   "camelcase",
+  "cancellable",
+  "cancelled",
+  "captcha",
   "cfn",
+  "cfnlambda",
+  "chatbot",
+  "chatbots",
+  "chdir",
   "checkbox",
-  "chown",
+  "checkmark",
+  "chokidar",
+  "cicd",
   "cidr",
   "cidrs",
   "circleci",
   "cloudformation",
   "cloudfront",
+  "cloudwatch",
+  "cname",
   "codebase",
   "codegen",
   "codepipeline",
+  "codesuite",
   "cognito",
+  "colocate",
   "columnify",
+  "commitlint",
   "commonjs",
   "cors",
-  "createCipheriv",
+  "createcipheriv",
   "creds",
   "cron",
+  "csproj",
+  "curr",
+  "cygwin",
+  "darwin",
   "datasource",
   "datastore",
+  "datepicker",
+  "datetime",
+  "decipheriv",
   "decrypt",
   "defs",
+  "denylist",
+  "devtool",
+  "devtools",
   "dict",
   "diffable",
+  "dirpath",
+  "distutils",
+  "djava",
+  "dockerfile",
+  "doctype",
   "dotenv",
+  "dotnet",
+  "dotnetcore3",
+  "dotnetcore31",
   "durations",
   "dynamodb",
+  "ecluster",
+  "ecma",
+  "ediscovery",
+  "ehttp",
+  "ejs",
   "elasticsearch",
+  "emacs",
+  "endian",
+  "enode",
+  "entrypoint",
+  "entrypoints",
   "enum",
+  "enums",
+  "epath",
   "errno",
+  "esri",
   "execa",
+  "executables",
   "extensibility",
   "extentions",
   "facebook",
   "fargate",
   "filenames",
+  "filepath",
+  "filepaths",
   "filesystem",
   "firebase",
   "foreach",
   "formatter",
   "frontend",
+  "frontends",
+  "fsext",
   "func",
   "funcs",
   "generatable",
+  "generategraphqlpermissions",
```

Comment:
should this be included?

Reply:
I see this is a feature flag converted to lowercase

Reply:
Looking at this again, I think it can be removed. It's used as a feature flag across the application. In test files, it is referenced in all lower case. In project files, it is referenced in camel case. I suspect the test files' usage is a mistake.